clickhouse/client.cpp

```diff
@@ -57,8 +57,11 @@ struct ClientInfo {
 };
 
 std::ostream& operator<<(std::ostream& os, const ClientOptions& opt) {
-    os << "Client(" << opt.user << '@' << opt.host << ":" << opt.port
-       << " ping_before_query:" << opt.ping_before_query
+    os << "Client(" << opt.user << '@' << "{ ";
+    for (const ClientOptions::HostPort& hp : opt.hosts_ports) {
+        os << hp.host << ":" << opt.port << ",";
```

Comment:
Try to make output so that if you have just one host it is the same as before your changes

Reply:
[Done](https://github.com/ManagedDatabases/clickhouse-cpp/pull/1/files#diff-93804ec8d3d55b5bfb169a28078e1183c38648da22398e8697c825977e1d3f20R525) and also fixed other errors (like forgotten `}`, etc)